OpenRA.Mods.Common/AI/Squad.cs

```diff
@@ -77,6 +76,12 @@ public bool TargetIsValid
 			get { return Target.IsValidFor(Units.FirstOrDefault()) && !Target.Actor.HasTrait<Husk>(); }
 		}
 
+		public bool TargetIsInVisible
+		{
+			get { return Target.Type == TargetType.Actor && !TargetActor.HasTrait<FrozenUnderFog>()
+				 && !Units.FirstOrDefault().Owner.Shroud.IsTargetable(TargetActor); }
```

Comment:
This probably wants to be `Bot.Player.PlayerActor.Owner.Shroud.IsTargetable(TargetActor);` otherwise it will crash when there are no units and the trait look is a bit expensive here.